### ISSUE TITLE
[New build - DEFAULT] 24w30b: Survival Improvements

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#5902]
Some improvements related to survival.

Changes:
- Solar, wind and Arc Furnace power level have been tweaks. Now several 1m solar/wind will be necessary to power a single furnace.
- Ore will take significantly less volume, at now 25l per ore.
- Scanner tweaked: now the metal type will be shown as letter

Hotfixes:
#5895 Crucible draws power

**Comment by Dwarf-LordPangolin** [comment]
Just wanted to let you know that the changes to the scanner have been a massive QOL improvement; I couldn't find silver before at all, and now I can right away! I did notice that copper was shown as Cp, where the element symbol is Cu. Also, the duration for which the scanner shows minerals isn't very long; might want to increase that a few more seconds at some point in the future.

**Comment by asanagisae** [comment]
It's good that ores now have letters to confirm what they are. Unfortunately, they've lost all color, so you _have to_ read the letters to know what they are.

![image](https://github.com/user-attachments/assets/4e79fa8b-c1ee-4517-91a1-77b150dcdfe0)

Both would be better, to have color draw your eyes and letters to confirm.

**Comment by ProPeach** [comment]
I agree, losing the colours is a bit of a downgrade. Colour *and* element label would be ideal for readability and accuracy. The size of the text label is about perfect though, thank you!

Also the labelling, I think if you're gonna go with elemental naming then it should be consistent. So Copper is Cu, Coal is C and then you can have Cobalt as Co. The current Cp label would be for copernicium!

These three labels are all pretty similar, all having C in them, but when you also have the colour then you can easily see the difference between the patches as they will be coppery/brown, black and blue respectively

**Comment by tsunamayo** [comment]
Hi, I will fix Copper as Cu. For the color, I was not a fan of them, as they can be hard/impossible to see depending on the color of the terrain. So gold is good offender.
If you dont see the mineral name then you can repress again, text might show up. Yeah it is crappy, but thats the best I can do for now.
Cheers

**Comment by ProPeach** [comment]
Thanks for changing up the copper!

Would it be possible to give the coloured dots a dark/black outline? That would help a lot with visibility against similar colours like the example you give.

**Comment by tsunamayo** [comment]
@ProPeach I was just referring to the fact that sometime you get a letter and sometime not, this is a limitation on how I generate my mineral patches, I dont intend to change that. But yeah I will still need to iterate on that, believe I know nothing is really ever done... I will tweak again, yes I think adding an outline like for the text could help to read colors.

**Comment by tsunamayo** [comment]
I want to add that for example one thing that I dont like and that will need tweaking is that it shows underground patches too much. It would prefer to have the player to have to explore the surface more (with hovercraft humm) rather that go deep rock galactic 20m from the starting point. So I will likely have to adjust that...

**Comment by ProPeach** [comment]
Sounds good! That might push the player towards a vehicle too. Maybe the player has to wait till they can build a ship-mounted scanner until they can detect the deeper (larger?) deposits.
